hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java

```diff
@@ -294,6 +294,7 @@ private OzoneConsts() {
   public static final String USED_NAMESPACE = "usedNamespace";
   public static final String QUOTA_IN_BYTES = "quotaInBytes";
   public static final String QUOTA_IN_NAMESPACE = "quotaInNamespace";
+  public static final String BUCKET_ENCRYPTION_KEY_NAME = "bucketEncryptionKey";
```

Comment:
It turns out we can reuse existing constant:

https://github.com/apache/ozone/blob/a66ad7cfc027f70dee51692050e73cb16bc960ed/hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java#L339

Reply:
Thank you @adoroszlai for pointing this out! I have reused the existing constant!